ext/reflection/php_reflection.stub.php

```diff
@@ -8,8 +8,8 @@ class ReflectionException extends Exception
 
 class Reflection
 {
-    /** @return string[] */
```

Comment:
I removed a bunch of these syntaxes... I'm not sure how much we need them, but in case we want them, I'll revert these changes